src/phosphor-rbmc-data-sync-mgr_main.cpp

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#include "common_types.hpp"
+
+#include <phosphor-logging/lg2.hpp>
+#include <sdbusplus/bus.hpp>
+#include <sdeventplus/event.hpp>
+
+int main()
+{
+    auto eventLoopRet = 0;
+    try
+    {
+        lg2::info("Welcome to RBMC Data-Sync");
```

Comment:
It’s not required. It’s added to perform a sanity check. Add a "TODO/FIXME" comment to indicate that this trace should be removed.